mypy.ini

```diff
@@ -87,9 +84,6 @@ exclude = (?x)
    |tests/state/test_v2.py
    |tests/storage/test_background_update.py
    |tests/storage/test_base.py
-   |tests/storage/test_client_ips.py
-   |tests/storage/test_database.py
-   |tests/storage/test_event_federation.py
```

Comment:
Is this a related change? Perhaps the datastore annotations are needed to get these tests to typecheck?

Reply:
This is related to reduce the number of exceptions for mypy.
These test are ok at the moment and does not create an error in the current status.  But they are certainly not complete with annotations. Like most tests.